packages/desktop-electron/index.js

```diff
@@ -165,6 +165,21 @@ async function createWindow() {
     win.webContents.send('set-socket', { name: serverSocket });
   });
 
+  // hit when middle-clicking buttons or <a href/> with a target set to _blank
+  // always deny, optionally redirect to browser
+  win.webContents.setWindowOpenHandler(({ url }) => {
+    processUrl(url);
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ah.. it's still pretty difficult to read. Could we instead just do this? I know it's a couple more lines, but then the code is immediately readable

```suggestion
    if (isExternalUrl(url)) {
        shell.openExternal(url);
    }
```